package.json

```diff
@@ -8,6 +8,7 @@
   "license": "MPL-2.0",
   "scripts": {
     "compile": "babel-compile -p taskcluster src:lib test-src:test",
+    "install": "babel-compile -p taskcluster src:lib test-src:test",
```

Comment:
good catch!  Maybe we could make the install script call the compile script as well, like we do for pretest.

Reply:
Fixed.